.github/workflows/quest.yml

```
@@ -15,7 +15,12 @@ on:
 
 jobs:
   import:
-    # if: github.event.label.name == join(fromJson('[":", "world_map", ":", " reQUEST"]'), '') || github.event_name == 'workflow_dispatch'
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      (contains(github.event.label.name, 'world_map') && contains(github.event.label.name, 'reQUEST')) ||
+      (contains(github.event.label.name, 'pushpin') && contains(github.event.label.name, 'seQUESTered')) ||
+      (contains(github.event.issue.labels.*.name, 'world_map') && contains(github.event.issue.labels.*.name, 'reQUEST')) ||
+      (contains(github.event.issue.labels.*.name, 'pushpin') && contains(github.event.issue.labels.*.name, 'seQUESTered'))
```

I think we can safely simplify this (and it will work in the repos that prefer not using the emojis in the labels):

```suggestion
      contains(github.event.label.name, 'reQUEST') ||
      contains(github.event.label.name, 'seQUESTered') ||
      contains(github.event.issue.labels.*.name, 'reQUEST') ||
      contains(github.event.issue.labels.*.name, 'seQUESTered')
```

---

Nice, check it out now!